hibernate-osgi/src/test/java/org/hibernate/osgi/test/OsgiIntegrationTest.java

```diff
@@ -99,13 +100,7 @@
 						.frameworkUrl( paxExamEnvironment.getProperty( "org.ops4j.pax.exam.container.karaf.distroUrl" ) )
 						.karafVersion( paxExamEnvironment.getProperty( "org.ops4j.pax.exam.container.karaf.version" ) )
 						.name( "Apache Karaf" )
-						.unpackDirectory(
-								new File(
-										paxExamEnvironment.getProperty(
-												"org.ops4j.pax.exam.container.karaf.unpackDir"
-										)
-								)
-						)
+						.unpackDirectory( new File( paxExamEnvironment.getProperty( "org.ops4j.pax.exam.container.karaf.unpackDir" ) ) )
```

Comment:
I think the whole change to this file can be reverted.

Reply:
Yes, surprisingly, we don't need anymore to touch it.